packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.instance-disable-api-termination.ts

```diff
@@ -0,0 +1,42 @@
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+
+const app = new cdk.App();
+
+class TestStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const vpc = new ec2.Vpc(this, 'VPC');
+    const securityGroup = new ec2.SecurityGroup(this, 'IntegSg', {
+      vpc,
+      allowAllIpv6Outbound: true,
+    });
+
+    const instance = new ec2.Instance(this, 'Instance', {
+      vpc,
+      securityGroup,
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.T3A, ec2.InstanceSize.MICRO),
+      machineImage: new ec2.AmazonLinuxImage({ generation: ec2.AmazonLinuxGeneration.AMAZON_LINUX_2023 }),
+      ebsOptimized: true,
+      disableApiTermination: true,
+      blockDevices: [{
+        deviceName: '/dev/xvda',
+        volume: ec2.BlockDeviceVolume.ebs(8, {
+          volumeType: ec2.EbsDeviceVolumeType.GP3,
+          encrypted: true,
+          deleteOnTermination: true,
+        }),
+      }],
+    });
+
+    instance.connections.allowFromAnyIpv4(ec2.Port.icmpPing());
+  }
+}
+
+const testCase = new TestStack(app, 'integ-ec2-instance-disable-api-termination');
+
+new IntegTest(app, 'instance-disable-api-termination', {
+  testCases: [testCase],
+});
```

Comment:
Could you add a `awsApiCall` assertion to make sure that the property is being set? See [INTEGRATION_TESTS](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md#assertions) and [DescribeInstanceAttribute](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstanceAttribute.html)

Reply:
Added assertion